src/dashboard/Data/Browser/DataBrowser.react.js

```diff
@@ -191,11 +199,12 @@ export default class DataBrowser extends React.Component {
         }
         e.preventDefault();
         break;
-      case 37: // Left
+      case 37: // Left - standalone & with ctrl/meta
```

Comment:
Could you clarify the comment for future developers?

Reply:
Comment updated 👍